### PR TITLE
doc: Explain how to log 404s in django

### PR DESCRIPTION
**Pull request by untitaker**
lmk if you need this for other frameworks.

### CODE REVIEW

File: src/collections/_documentation/platforms/python/django.md [hunk 1]
```diff
@@ -66,3 +66,32 @@ The following keyword arguments can be passed to `DjangoIntegration()`:
 
 The user feedback feature can be used with this integration.  For more information
 see [User Feedback]({% link _documentation/enriching-error-data/user-feedback.md %}?platform=django).
+
+## Reporting other status codes
+
+In some situations it might make sense to report 404s and other errors besides 500 to Sentry. This can be achieved by writing your own Django view for those status codes. For example:
```
Comment (lobsterkatie) [inline src/collections/_documentation/platforms/python/django.md]: > other errors besides 500

Are 500 errors literally all we capture, or by default do we capture all 500-level errors (501, 502, etc)?

Comment (untitaker) [reply src/collections/_documentation/platforms/python/django.md]: Only 500. We don't have some code like `if status_code == 500` in there, but rely on Django to tell us what is an unhandled exception.

File: src/collections/_documentation/platforms/python/django.md [hunk 2]
```diff
@@ -66,3 +66,32 @@ The following keyword arguments can be passed to `DjangoIntegration()`:
...
+# views.py
+
+from django.http import HttpResponseNotFound
+from sentry_sdk import capture_message
+
+
+def my_custom_page_not_found_view(*args, **kwargs):
+    capture_message("Page not found!", level="error")
+
+    # return any response here
```
Comment (lobsterkatie) [inline src/collections/_documentation/platforms/python/django.md]: How would you feel about this:

```
# return any response here, for ex:
return HttpResponseNotFound("Not found")
```
for the last two lines?

Comment (untitaker) [reply src/collections/_documentation/platforms/python/django.md]: can do

Comment (untitaker) [reply src/collections/_documentation/platforms/python/django.md]: done

File: src/collections/_documentation/platforms/python/django.md (hunk 1 above)
Comment (untitaker) [reply src/collections/_documentation/platforms/python/django.md]: your call on this one, no opinion

**Comment by untitaker** [comment]
@lobsterkatie pls review again

File: src/collections/_documentation/platforms/python/django.md (hunk 1 above)
Comment (lobsterkatie) [reply src/collections/_documentation/platforms/python/django.md]: Yeah, I like "besides general unhandled exceptions" better - it's closer to the full story without getting too deep into the weeds.

**Comment by lobsterkatie** [comment]
With that small change to the 500/unhandled errors language, I think it's good to go.

(Will be traveling part of tomorrow (before working the rest of the day remotely), so if I don't see the update right away, feel free to ping me in slack.)

**Pull request merged**